apps/edxapp/templates/cms/job_01_collectstatic.yml.j2

```diff
@@ -52,25 +52,28 @@ spec:
             - "-c"
             - cp -r /config/* /tmp/config &&
               cp -rL /tmp/configmap-cms/* /tmp/config/cms/ &&
-              cp /tmp/secret/credentials.vault.yml /tmp/config/cms/secrets.yml
+              cp /tmp/secret/credentials.vault.yml /tmp/config/cms/secrets.yml &&
+              cp -rL /edx/app/edxapp/staticfiles/* /tmp/staticfiles/
```

Comment:
We should add a long comment to make sure we remember the why and how of this hack + an issue to try to make a PR to edX and solve it at the root of the problem?

Reply:
Working on it.